src/sonic-yang-models/yang-models/sonic-mclag.yang

```diff
@@ -0,0 +1,171 @@
+module sonic-mclag {
+	namespace "http://github.com/Azure/sonic-mclag";
+	prefix smclag;
+	yang-version 1.1;
+
+
+    import ietf-yang-types {
```

Comment:
can you check the space to be consistent? 4 spaces

Reply:
done